src/core/report/report-generator.ts

```
@@ -0,0 +1,132 @@
+   * Generates a report from evaluation results
+   */
+  generateReport(results: EvaluationReport): string {
+    switch (this.config.outputFormat) {
+      case 'json':
+        return this.generateJsonReport(results);
+      case 'markdown':
+        return this.generateMarkdownReport(results);
+      case 'text':
+        return this.generateTextReport(results);
+      default:
+        const format = this.config.outputFormat as string;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)